src/main/java/org/folio/inventory/domain/view/request/StoredRequestView.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.inventory.domain.view.request;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import org.joda.time.DateTime;
+
+public class StoredRequestView {
```

Comment:
What is a stored request view?

Reply:
It is a circulation request.

Reply:
Apologies for asking a vague question. 

What do `stored` and `view` mean in this context? 

Does inventory store it's own representation of the request?

Reply:
The stored means the request from storage but not from `mod-circulation`. And by view I mean that the class contains all the properties as raw key-values and some delegating methods to update/read these properties.

Do you think it should be renamed?

Reply:
> The stored means the request from storage but not from `mod-circulation`. And by view I mean that the class contains all the properties as raw key-values and some delegating methods to update/read these properties.
> 
> Do you think it should be renamed?

Thank you. Those aspects don't come across naturally for me from the name alone. Personally, I think it could be called a `Request` without losing much of the descriptive power.

I think the view aspect is (hopefully) an internal modelling, if it doesn't affect the behaviour or the client API then I think it can be kept private and not part of the name.

The storage aspect is interesting, I think that could be expressed in the class that is responsible for fetching it from storage and updating it in storage. Unless, there are specific aspects of the storage representation that this class leaks out to the rest of the module?

Does that make sense?

Reply:
Class has been renamed to `Request` and `RequestRepository` has been created which uses this class.